### PR TITLE
fix: tweaking throttle value

### PR DESCRIPTION
**Pull request by anipendakur-okta**
OKTA-321999

## Description:
There were two competing throttled submits hence the race condition. Tweaking the throttle should fix the issue.

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Reviewers:
@magizh-okta @gowthamidommety-okta 

### Issue:

- [OKTA-321999](https://oktainc.atlassian.net/browse/OKTA-321999)

### CODE REVIEW

**Review by magizh-okta — APPROVED** [review]
LGTM 🚢 can you please update the commit description to be more accurate. Just say that updating the throttle value to be the same as in base form. Over here there is no competing throttles.

File: src/views/mfa-verify/SendEmailAndVerifyCodeForm.js [hunk 1]
```diff
@@ -60,8 +60,8 @@ const SendEmailAndVerifyCodeFormVerifyEmailCodeForm = Form.extend({
   initialize: function () {
     Form.prototype.initialize.apply(this, arguments);
 
-    //Added thorttle to prevent keyboard enter trigger multipele API calls
-    this.handleSubmit = _.throttle(this.handleSubmit, 100, { leading: false });
+    // Added throttle to prevent keyboard enter trigger multiple API calls
+    this.handleSubmit = _.throttle(this.handleSubmit, 200, { leading: false });
```
Comment (pradeepdewda-okta) [inline src/views/mfa-verify/SendEmailAndVerifyCodeForm.js:64]: is it going to work on slow internet connection? Also want to get some insight how increasing the throttle works and won't break in future or with varied internet bandwidths

Comment (gowthamidommety-okta) [reply src/views/mfa-verify/SendEmailAndVerifyCodeForm.js:64]: I'm also wondering the same

File: src/models/Settings.js [hunk 2]
```diff
@@ -78,7 +78,6 @@ export default Model.extend({
     'features.trackTypingPattern': ['boolean', false, false],
     'features.redirectByFormSubmit': ['boolean', false, false],
     'features.useDeviceFingerprintForSecurityImage': ['boolean', false, true],
-    'features.hideDefaultTip': ['boolean', false, true],
```
Comment (pradeepdewda-okta) [inline src/models/Settings.js:81]: why this was removed ?

File: src/views/primary-auth/PrimaryAuthForm.js [hunk 3]
```diff
@@ -111,7 +111,7 @@ export default Form.extend({
       label: loc('primaryauth.username.placeholder', 'login'),
       'label-top': true,
       explain: () => {
-        if (this.settings.get('features.hideDefaultTip') && !this.isCustomized('primaryauth.username.tooltip')) {
```
Comment (pradeepdewda-okta) [inline src/views/primary-auth/PrimaryAuthForm.js:114]: does this have any relation with throttle issue ?

**Comment by anipendakur-okta** [comment]
This PR is not required anymore, closing it..

**Pull request closed without merging**